src/test/sighash_tests.cpp

```diff
@@ -124,12 +124,12 @@ void static RandomTransaction(CMutableTransaction &tx, bool fSingle) {
             } else {
                 pourtx.vpub_new = insecure_rand() % 100000000;
             }
-            RandomScript(pourtx.scriptPubKey);
-            RandomScript(pourtx.scriptSig);
+
             pourtx.anchor = GetRandHash();
             pourtx.serials[0] = GetRandHash();
             pourtx.serials[1] = GetRandHash();
             pourtx.ephemeralKey = GetRandHash();
```

Comment:
(This doesn't matter because we're just testing encoding/signature stuff right, i.e. it's never actually used as a public key?)